backend/app/src/main/resources/db/migration/V1__Initial_ddl_script.sql

```diff
@@ -43,66 +43,56 @@ CREATE TABLE IF NOT EXISTS control (
-  bus_owner_name                        varchar(255),
-  bus_owner_type                        varchar(255),
-  bus_owner_type_other                  varchar(255),
-  fac_building_name                     varchar(100),
-  facility_hds_details_additional_info  varchar(255),
-  physical_addr1                        varchar(255),
-  physical_addr2                        varchar(255),
-  physical_addr3                        varchar(255),
-  physical_addr4                        varchar(255),
-  physical_city                         varchar(255),
-  physical_province                     varchar(255),
-  physical_pcode                        varchar(255),
-  physical_country                      varchar(255),
-  phys_addr_is_private                  varchar(1),
-  mail_addr1                            varchar(255),
-  mail_addr2                            varchar(255),
-  mail_addr3                            varchar(255),
-  mail_addr4                            varchar(255),
-  mail_city                             varchar(255),
-  mail_bc                               varchar(255),
-  mail_pcode                            varchar(255),
-  mail_country                          varchar(255),
-  mail_addr_is_private                  varchar(1),
+  do_not_load_flag VARCHAR(10),
+  stakeholder VARCHAR(20),
+  stakeholder_id VARCHAR(19),
+  hds_ipc_id VARCHAR(19),
+  hds_cpn_id VARCHAR(19),
+  hds_provider_identifier1 VARCHAR(50),
+  hds_provider_identifier2 VARCHAR(50) ,
+  hds_provider_identifier3 VARCHAR(50),
+  hds_provider_identifier_type1  VARCHAR(50),
+  hds_provider_identifier_type2  VARCHAR(50),
+  hds_provider_identifier_type3  VARCHAR(50),
+  hds_msp_facility_number VARCHAR(50),
+  hds_type VARCHAR(400),
+  hds_sub_type VARCHAR(50),
+  hds_name VARCHAR(100),
+  hds_preferred_name_flag VARCHAR(1),
+  hds_email  VARCHAR(50),
+  hds_website  VARCHAR(50),
+  hds_bus_tel_area_code VARCHAR(15), 
+  hds_bus_tel_number VARCHAR(30), 
+  hds_tel_extension VARCHAR(15),
+  hds_cell_area_code VARCHAR(15), 
+  hds_cell_number VARCHAR(30),
+  hds_fax_area_code VARCHAR(15), 
+  hds_fax_number VARCHAR(30),
+  pcn_service_delivery_type VARCHAR(255),
+  pcn_clinic_type VARCHAR(255), 
+  pcn_pci_flag VARCHAR(1), 
+  source_status VARCHAR(50),
+  pcn_clinic_status VARCHAR(255), 
+  hds_effective_start_date VARCHAR(255),
+  fac_address_unit VARCHAR(100),
+  fac_building_name VARCHAR(100), 
+  physical_addr1 VARCHAR(255),
+  physical_addr2 VARCHAR(255),
+  physical_addr3 VARCHAR(255),
+  physical_addr4 VARCHAR(255),
+  physical_city VARCHAR(255),
+  physical_province VARCHAR(255),
+  physical_pcode VARCHAR(255),
+  physical_country VARCHAR(255), 
+  physical_addr_prps_type_cd VARCHAR(255),    
+  mail_addr1 VARCHAR(255),
+  mail_addr2 VARCHAR(255),
+  mail_addr3 VARCHAR(255),
+  mail_addr4 VARCHAR(255),
+  mail_city VARCHAR(255),
+  mail_bc VARCHAR(255),
+  mail_pcode VARCHAR(255),
+  mail_country VARCHAR(255),
   created_by VARCHAR(30) DEFAULT CURRENT_USER,
```

Comment:
I guess it should not default to current user. It should be non-nullable though. Having non-default value forces application code to provide reasonable value and everyone is aware of the value.

Reply:
Removed default values from database script. Default values are set in the backend code.